example/qds_spark_python_example.py

```diff
@@ -1,5 +1,5 @@
 """
-This is a sample code used for submitting a Spark script (SparkCommand) and getting the result.
+This is a sample code used for submitting a Spark Python/R/Scala command and getting the result.
```

Comment:
Suggestion:

The following code example submits a Spark program written in either Scala, Python, SQL, or R using SparkCommand command type. The input program/script is made available using a local file path and the results are written to a temporary file. 

REST API reference: https://docs.qubole.com/en/latest/rest-api/command_api/submit-a-spark-command.html